src/Wizard/index.test.jsx

```diff
@@ -56,4 +75,26 @@ describe('Wizard', () => {
 
     expect(getByText('Step 1')).toBeInTheDocument();
   });
+
+  it('can not click on "Finish" button when second step has an error, goes to first step by clicking on "Previous" then return to second step', async () => {
```

Comment:
```suggestion
  it('cannot finish the Wazard when there is a validation error, but can change steps', async () => {
```

Test names should describe the case, not describe the details of the steps in place